execution.py

```diff
@@ -1,502 +1,502 @@
-        msg = "Disks are added to Diskpool"
-        if str(res) == '<Response [200]>':
-            LogCreat().logger_info.info(msg)
-        else:
-            self.verification(res.json(), msg)
-    def reclamination(self):
-        '''
-        This method verify reclamination of diskpool.
-        Arguments : None
-        Return (int): self.flag
-        '''
-        uri = "performancebytype/DiskPoolPerformance"
-        res = ILDC().do_ssy_details(uri, header=None)
-        if res.json() != []:
-            for _ in res.json():
-                for key,val in _.items():
-                    if key == 'PerformanceData' and val["BytesInReclamation"] == 0:
-                        self.flag = 1
-            time.sleep(60)
-        return self.flag
-    def get_server(self):
-        '''
-        This method used to get server details.
-        Arguments : None
-        Return: None
-        '''
-        uri = "servers"
-        res = ILDC().do_ssy_details(uri, header=None)
-        self.server_id = res.json()[0]['Id']
-        msg = 'Get server details'
-        if str(res) == '<Response [200]>':
-            LogCreat().logger_info.info(msg)
-        else:
-            self.verification(res.json(), msg)
-    def get_host(self):
-        '''
-        This method used to get host details.
-        Arguments : None
-        Return: None
-        '''
-        uri = 'hosts'
-        res = ILDC().do_ssy_details(uri, header=None)
-        msg = 'Get host details'
-        if str(res) == '<Response [200]>':
-            LogCreat().logger_info.info(msg)
-        else:
-            self.verification(res.json(), msg)
-    def test_disable_cap_opt_at_server(self):
-        '''
-        This method used to disable capacity optimization at
-        server level.
-        Arguments : None
-        Return: None
-        '''
-        uri = "servers/" + self.server_id
-        payload_dict = {
-            "Operation": "RemoveCapacityOptimizationDisks",
-            "Disks": self.co_disk,
-        }
-        res = ILDC().do_disable_capacity_optimization(uri, header=None, payload=payload_dict)
-        msg = "Capacity Optimization is disabled successfully at server level"
-        self.verification(res.json(), msg)
-    def delete_pool(self):
-        '''
-        This method used to delete diskpool.
-        Arguments : None
-        Return: None
-        '''
-        uri = "pools/" + self.pool_id
-        res = ILDC().do_pool_delete(uri)
-        msg = "Diskpool deleted successfully"
-        if str(res) == '<Response [200]>':
-            print(msg)
-            LogCreat().logger_info.info(msg)
-        else:
-            self.verification(res.json(), msg)
-    def test_create_virtual_disk(self, virtual_disk):
-        '''
-        This method used to create Virtual disk.
-        Arguments (str): virtual disk
-        Return: None
-        '''
-        uri = "virtualdisks"
-        vd_payload = {
```

Comment:
Can this be implement outside the script as a configuration file instead of hardcoding ?

Reply:
no its part of RestAPI we cant disturb that.